archinstall/default_profiles/desktops/cosmic.py

```diff
@@ -0,0 +1,21 @@
+from typing import List, Optional, Any, TYPE_CHECKING
+
+from archinstall.default_profiles.profile import ProfileType, GreeterType
+from archinstall.default_profiles.xorg import XorgProfile
+
+if TYPE_CHECKING:
+	_: Any
+
+class PlasmaProfile(XorgProfile):
```

Comment:
Nice work, but could we change the class name to not collide with the plasma profile?